scripts/heat/ping_template.sh

```diff
@@ -43,3 +43,18 @@ cat >> /usr/bin/ping_forever_neighbour << EOF
 EOF
 chmod +x /usr/bin/ping_forever_neighbour
 /usr/bin/ping_forever_neighbour &
+
+cat >> /usr/bin/cinder_test << EOF
+  #!/bin/sh
+
+  sudo mkfs -t ext4 -L cinder_volume /dev/vdb
```

Comment:
I think that by default any *first* attached volume will get that by default for our kvm hypervisor.

I believe you can set the device name explicitly, but I think kvm ignores it anyway:

```
root@d52-54-77-77-77-01:~ # openstack server add volume --device /dev/vdg test1 test5
root@d52-54-77-77-77-01:~ # openstack volume list
+--------------------------------------+--------------+--------+------+--------------------------------+
| ID                                   | Display Name | Status | Size | Attached to                    |
+--------------------------------------+--------------+--------+------+--------------------------------+
| 52302393-8455-4436-86f6-c77701f6e439 | test5        | in-use |    1 | Attached to test1 on /dev/vdd  |

```